### ISSUE TITLE
是不是没办法在 webstorm 使用又拍云？

### ISSUE THREAD

**Issue by ytahml** [dong4j/markdown-image-kit#23]
![image](https://github.com/user-attachments/assets/4dfbe901-f969-4138-8efa-b1dfbb010a21)

**Comment by dong4j** [comment]
不是 是因为压根还没有实现又拍云, 2.x 版本是一个全新版本, 除了几个常用的对象存储服务, 后续不再支持其他对象存储服务商, 你可以使用自定义或 PicList/PicGO 来中转到其他服务商
